docs/en/latest/plugins/loki-logger.md

```diff
@@ -28,92 +28,69 @@ description: This document contains information about the Apache APISIX loki-log
-
-### Example of default log format
+| Name | Type | Required | Default | Valid values | Description |
+|---|---|---|---|---|---|
+| endpoint_addrs | array[string] | True |  | | Loki API base URLs, such as `http://127.0.0.1:3100`. If multiple endpoints are configured, the log will be pushed to a randomly determined endpoint from the list. |
+| endpoint_uri | string | False | /loki/api/v1/push | | URI path to the Loki ingest endpoint. |
+| tenant_id | string | False | fake | | Loki tenant ID. According to Loki's [multi-tenancy documentation](https://grafana.com/docs/loki/latest/operations/multi-tenancy/#multi-tenancy), the default value is set to `fake` under single-tenancy. |
+| log_labels | object | False | {job = "apisix"} | | Loki log label. Support [Nginx variables](https://nginx.org/en/docs/varindex.html) and constant strings in values. Variables should be prefixed with a `$` sign. For example, the label can be `{"origin" = "apisix"}` or `{"origin" = "$remote_addr"}`. |
+| ssl_verify        | boolean       | False    | true | | If true, verify Loki's SSL certificates. |
+| timeout           | integer       | False    | 3000 | [1, 60000] | Timeout for the Loki service HTTP call in milliseconds.  |
+| keepalive         | boolean       | False    | true |  | If true, keep the connection alive for multiple requests. |
+| keepalive_timeout | integer       | False    | 60000 | >=1000 | Keepalive timeout in milliseconds.  |
+| keepalive_pool    | integer       | False    | 5       | >=1 | Maximum number of connections in the connection pool.  |
+| log_format | object | False    |          | | Custom log format in key-value pairs in JSON format. Support [APISIX variables](../apisix-variable.md) and [Nginx variables](http://nginx.org/en/docs/varindex.html) in values. |
+| name | string | False    | loki-logger | | Unique identifier of the Plugin for the batch processor. If you use [Prometheus](./prometheus.md) to monitor APISIX metrics, the name is exported in `apisix_batch_process_entries`. |
+| include_req_body       | boolean | False    | false | | If true, include the request body in the log. Note that if the request body is too big to be kept in the memory, it can not be logged due to NGINX's limitations. |
+| include_req_body_expr  | array[array]   | False    |  | | An array of one or more conditions in the form of [lua-resty-expr](https://github.com/api7/lua-resty-expr). Used when the `include_req_body` is true. Request body would only be logged when the expressions configured here evaluate to true. |
+| include_resp_body      | boolean | False    | false | | If true, include the response body in the log.  |
+| include_resp_body_expr | array[array]   | False    |  | | An array of one or more conditions in the form of [lua-resty-expr](https://github.com/api7/lua-resty-expr). Used when the `include_resp_body` is true. Response body would only be logged when the expressions configured here evaluate to true. |
```

Comment:
Wrong links: [lua-resty-expr](https://github.com/api7/lua-resty-expr)?
![image](https://github.com/user-attachments/assets/898fd244-4e86-4bf4-a202-5b45c50b326f)

Reply:
this link is right